src/test/kotlin/io/bazel/kotlin/integration/WriteWorkspace.kt

```diff
@@ -0,0 +1,461 @@
+package io.bazel.kotlin.integration
+
+import java.io.Closeable
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Files.exists
+import java.nio.file.Path
+import java.nio.file.StandardOpenOption
+import java.nio.file.StandardOpenOption.CREATE_NEW
+import java.nio.file.attribute.FileTime
+import java.time.Instant
+import kotlin.streams.toList
+
+object WriteWorkspace {
```

Comment:
The naming of this singleton is a wee bit weird - but I really like the dsl.

A few example usages in a kdoc here would be super handy, so someone isn't hunting down examples in the tests.